packages/next-app/src/app/daoadmin/action/[actionId]/(components)/ActionAttributeContent.tsx

```diff
@@ -1,37 +1,45 @@
 "use client";
 
 import { useRouter } from "next/navigation";
-import { HTMLProps } from "react";
 import * as React from "react";
 import { useForm } from "react-hook-form";
+import { useNetwork } from "wagmi";
 
 import { Button } from "#/components";
-import { toSlug, useAdminTools } from "#/contexts/AdminToolsContext";
+import { Input } from "#/components/Input";
+import { useAdminTools } from "#/contexts/AdminToolsContext";
+import { fetchExistingPool } from "#/utils/fetcher";
+import { toCamelCase } from "#/utils/formatStringCase";
 import { truncateAddress } from "#/utils/truncateAddress";
 
 export function ActionAttributeContent() {
-  const { register, handleSubmit } = useForm();
+  const { register, handleSubmit, watch } = useForm();
   // eslint-disable-next-line no-console
   const onSubmit = (data: unknown) => console.log(data);
   const { push } = useRouter();
   const { selectedAction } = useAdminTools();
+  const { chain } = useNetwork();
+  const [poolSymbol, setPoolSymbol] = React.useState<string>();
+  const [poolError, setPoolError] = React.useState<string>();
 
-  const Input = React.forwardRef<HTMLInputElement, HTMLProps<HTMLInputElement>>(
-    ({ label, ...rest }: React.HTMLProps<HTMLInputElement>, ref) => {
-      return (
-        <div>
-          <label className="mb-2 block text-sm font-bold text-gray-400">
-            {label}
-          </label>
-          <input
-            ref={ref}
-            {...rest}
-            className="selection:color-white box-border inline-flex h-[35px] w-full appearance-none items-center justify-center rounded-[4px] bg-blackA5 px-[10px] text-[15px] leading-none text-white shadow-[0_0_0_1px] shadow-blackA9 outline-none selection:bg-blackA9 hover:shadow-[0_0_0_1px_black] focus:shadow-[0_0_0_2px_black] disabled:bg-blackA9"
-          />
-        </div>
-      );
+  const poolId = watch("poolID");
+
+  React.useEffect(() => {
+    if (!poolId) {
+      setPoolSymbol("");
+      setPoolError("");
+      return;
     }
-  );
+    const couldInputBePoolId = /^(0x){1}[0-9a-f]{64}/i.test(poolId);
```

Comment:
I'd rename it to `isPoolId` or `isPoolIdLike` or `isPoolIdFormat`

Reply:
thanks! `isPoolIdFormat` is great